ui_testing/testcases/basic_tests/test03_orders.py

```diff
@@ -0,0 +1,139 @@
+from ui_testing.testcases.base_test import BaseTest
+from ui_testing.pages.login_page import Login
+from ui_testing.pages.article_page import Article
+from ui_testing.pages.testplan_page import TstPlan
+from ui_testing.pages.order_page import Order
+from ui_testing.pages.analyses_page import Analyses
+from parameterized import parameterized
+import re
+
+
+class OrdersTestCases(BaseTest):
+    def setUp(self):
+        super().setUp()
+        self.login_page = Login()
+        self.article_page = Article()
+        self.test_plan = TstPlan()
+        self.order_page = Order()
+        self.analyses_page = Analyses()
+        self.login_page.login(
+            username=self.base_selenium.username, password=self.base_selenium.password)
+        self.base_selenium.wait_until_page_url_has(text='dashboard')
+        self.order_page.get_orders_page()
+
+    def test01_archive_order(self):
+        """
+            New: Orders: Archive
+            The user cannot archive an order unless all corresponding analysis are archived
+            LIMS-3425
+        :return:
+        """
+        order_row = self.order_page.get_random_order_row()
+        self.order_page.click_check_box(source=order_row)
+
+        analysis_number_value = self.order_page.get_row_cell_text_related_to_header(
+            order_row, 'Analysis No.')
```

Comment:
```suggestion
            order_row, 'Analysis No')
```
I can see that in staging.1lims.com, it doesn't have `.` at the end of the word